packages/map-popup/i18n/en-US.yml

```diff
@@ -3,7 +3,8 @@ otpUi:
     availableBikes: "Available bikes: {value}"
     availableDocks: "Available docks: {value}"
     floatingBike: "Free-floating bike: {name}"
-    floatingCar: "{company} {name}"
+    floatingCar: "Car: {name}"
```

Comment:
Should this change apply to other languages too?

Reply:
I made the change backwards compatible so I don't have to make all the translations. I don't feel confident translating for all those languages